draft-ietf-lpwan-schc-over-lorawan.md

```diff
@@ -120,8 +118,8 @@ This section contains a short overview of Static Context Header Compression
 
 It defines:
 
-1. Compression mechanisms to avoid transport of known data by both
-   sender and receiver over the air. Known data are part of the "context"
+1. Compression mechanisms to avoid transport over the air of known data by both
+   sender and receiver. Known data is part of the "context".
```

Comment:
This is now conflicting with Dominique proposition, which sound better to me
```
 Compression mechanisms to avoid transporting information known by both
  sender and receiver over the air. Known information are part of the "context"
```
Do you agree with it ?

Reply:
yes, it sounds better indeed. Do you want me to update the PR?

Reply:
if you have time yes please, otherwise I'll do it myself :)